app/controllers/quizzes/quizzes_controller.rb

```diff
@@ -906,6 +906,7 @@ def start_quiz!
   def take_quiz
     return unless quiz_submission_active?
     @show_embedded_chat = false
+    @allowed_filetypes = SettingsService.get_settings(object: 'school', id: 1)['allowed_filetypes']
```

Comment:
Can we do something to prevent "magic numbers" in code?  Hard coding id 1 could be problematic.

Reply:
It looks like everywhere we are calling `SettingsService.get_settings(...)` we are hard coding an `id` of 1. If I recall correctly, this just essentially corresponds to `Account.first.id`. I can change it to use the `Account` call so it's more clear what this is doing, but given how many times this appears in the code like this already, I'm not sure what the correct way forward should be... Thoughts, @dneighbors?